src/tests/system/pytest.ini

```diff
@@ -2,6 +2,13 @@
 addopts = --strict-markers
 testpaths = tests
 markers =
-    slow: marks tests as slow (deselect with '-m "not slow"')
+    authentication: marks authentication test
+    authorization: marks authorization test
+    cache: marks any cache related test
     contains_workaround_for(gh=...,bz=...): test requires workaround for an existing bug
+    identity: marks identity, lookups test
+    schema: marks schema test
```

Comment:
IMHO the tests with this mark don't do anything with schema, they don't even need extended schema. How do you image to use this marker?

Reply:
Any test that requires changes to the directory schema or tests the directory schema attributes and values. We have tests that are explicit to rfc2307 and then to rfc2307bis . 

They will be some overlap in tests suites, ldap password policy, can  go in the authentication marker but because it's changing values in the LDAP schema I felt that it also belongs here.

Reply:
Schema is vague here. "Changes to the directory schema" and setting what kind of schema is SSSD using is using for password policy are very different things. But ok.